tests/pipelines/test_pipelines_zero_shot_audio_classification.py

```diff
@@ -27,6 +27,8 @@ class ZeroShotAudioClassificationPipelineTests(unittest.TestCase):
     # and only CLAP would be there for now.
     # model_mapping = {CLAPConfig: CLAPModel}
 
+    # TODO: fix me (ydshieh)
```

Comment:
well, probably @lhoestq too

Reply:
The issue comes from the CI using `datasets @ git+https://github.com/huggingface/datasets@269fcd31a2e759c65ffd5952ecef13e6a0d92574` instead of the latest release of `datasets` (see "Show installed libraries and their versions" section in CircleCI)

This must be related to the CI runs I did on a dedicated branch to test `transformers` against `datasets@main`